jenkins_pipelines/environments/common/pipeline-pull-request-light.groovy

```diff
@@ -0,0 +1,208 @@
+        arch= 'x86_64'
+        built = false
+        deployed = false
+        tests_passed = false
+        sumaform_backend = 'libvirt'
+        terraform_bin = '/usr/bin/terraform'
+        terraform_bin_plugins = '/usr/bin'
+        service_pack_migration = false
+//        terracumber_gitrepo = 'https://github.com/uyuni-project/terracumber.git'
+        terracumber_gitrepo = 'https://github.com/maximenoel8/terracumber.git'
+        terracumber_ref = 'support_tfvars'
+        terraform_init = true
+        rake_namespace = 'cucumber'
+        rake_parallel_namespace = 'parallel'
+        jenkins_workspace = '/home/jenkins/jenkins-build/workspace/'
+        pull_request_repo = 'https://github.com/uyuni-project/uyuni.git'
+        builder_api = 'https://api.opensuse.org'
+        build_url = 'https://build.opensuse.org'
+        builder_project = 'systemsmanagement:Uyuni:Master:PR'
+        source_project = 'systemsmanagement:Uyuni:Master'
+        sumaform_tools_project = 'systemsmanagement:sumaform:tools'
+        test_packages_project = 'systemsmanagement:Uyuni:Test-Packages:Pool'
+        build_repo = 'openSUSE_Leap_15.4'
+        environment_workspace = null
+        url_prefix="https://ci.suse.de/view/Manager/view/Uyuni/job/${env.JOB_NAME}"
+        env.common_params = ''
+        fqdn_jenkins_node = sh(script: "hostname -f", returnStdout: true).trim()
+        env_number = 2
+        try {
+            stage('Checkout CI tools') {
+                ws(environment_workspace){
+                    if(must_test) {
+                        git url: terracumber_gitrepo, branch: terracumber_ref
+                        dir("susemanager-ci") {
+                            checkout scm
+                        }
+
+                        // Define test environment parameters
+                        env.resultdir = "${WORKSPACE}/results"
+                        env.resultdirbuild = "${resultdir}/${BUILD_NUMBER}"
+                        env.tf_file = "susemanager-ci/terracumber_config/tf_files/PR-TEST-main.tf"
+                        tfvariables_file  = 'susemanager-ci/terracumber_config/tf_files/variables/PR-TEST-variable.tf'
+                        tfvars_manager43 = 'susemanager-ci/terracumber_config/tf_files/tfvars/PR-TEST-manager43.tfvars'
+                        tfvars_nuremberg = 'susemanager-ci/terracumber_config/tf_files/tfvars/PR-TEST-NUE-ENVS.tfvars'
+                        env.common_params = "--outputdir ${resultdir} --tf ${tf_file} --gitfolder ${resultdir}/sumaform --tfvariables_file=${tfvariables_file} --tfvars_files=${tfvars_nuremberg} --tfvars_files=${tfvars_manager43}"
+
+                        if (params.terraform_parallelism) {
+                            env.common_params = "${env.common_params} --parallelism ${params.terraform_parallelism}"
+                        }
+
+                        // Clean up old results
+                        sh "if [ -d ${resultdir} ];then ./clean-old-results -r ${resultdir};fi"
```

Comment:
Being able to have if statements at Groovy level, I don't see a good reason to delegate it to bash.